charts/kueue/tests/manager_test.yaml

```diff
@@ -0,0 +1,15 @@
+suite: test manager deployment
+templates:
+  - manager/manager.yaml
+  # This is required for the annotation `charts.kueue.x-k8s.io/config-checksum`
```

Comment:
Please adjust the comment

Reply:
Would you mind providing more details about how to adjust? Thanks.

Reply:
Without this line, the following line will fail.

https://github.com/kubernetes-sigs/kueue/blob/6bbcb6a9e2c313dbf4a6deb05bf35f4136b829af/charts/kueue/templates/manager/manager.yaml#L18

Reply:
As I see now, thanks for clarification